src/MyTested.AspNetCore.Mvc.Controllers.Attributes/ControllerActionAttributesTestBuilderExtensions.cs

```diff
@@ -345,6 +345,57 @@ public static class ControllerActionAttributesTestBuilderExtensions
             return actualBuilder.AttributesTestBuilder;
         }
 
+        /// <summary>
+        /// Tests whether the collected attributes contain <see cref="MiddlewareFilterAttribute"/>.
+        /// </summary>
+        /// <param name="controllerActionAttributesTestBuilder">
+        /// Instance of <see cref="IControllerActionAttributesTestBuilder{TAttributesTestBuilder}"/> type.
+        /// </param>
+        /// <param name="configurationType">A type which configures a middleware pipeline.</param>
+        /// <returns>The same attributes test builder.</returns>
+        public static TAttributesTestBuilder SpecifyingMiddleware<TAttributesTestBuilder>(
+            this IControllerActionAttributesTestBuilder<TAttributesTestBuilder> controllerActionAttributesTestBuilder,
+            Type configurationType)
+            where TAttributesTestBuilder : IControllerActionAttributesTestBuilder<TAttributesTestBuilder>
+            => controllerActionAttributesTestBuilder
+                .SpecifyingMiddleware(middleware => 
+                    middleware.WithType(configurationType));
```

Comment:
The `middleware` part of the expression should be on the line above. Check the other builder for an example.